spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/mapper/ColumnMapper.scala

```diff
@@ -23,7 +23,7 @@ import scala.reflect.ClassTag
   * }}}
   */
 trait ColumnMapper[T] extends Serializable {
-  def columnMap(tableDef: TableDef): ColumnMap
+  def columnMap(tableDef: TableDef, aliases: Map[String, String] = Map.empty): ColumnMap
```

Comment:
It would be nice, but I don't see how it would work in the case when functions are used - functions, like TTL, write time or count are not columns.